### PR TITLE
Improve the `fsaa` setting description

### PR DESCRIPTION
**Pull request by Calinou**
This clarifies that Minetest needs to be restarted to see any changes.

See https://github.com/minetest/minetest/issues/8459.

## To do

This PR is Ready for Review.

PS: Do I have to update `minetest.conf.example` and `settings_translation_file.cpp` myself? I couldn't get the generator to work locally.

## How to test

Go to the Advanced Settings menu and check the FSAA description.

### CODE REVIEW

**Comment by paramat** [comment]
> Do I have to update minetest.conf.example and settings_translation_file.cpp myself?

No, we auto-generate those occasionally, so your work would be wasted effort.

File: builtin/settingtypes.txt [hunk 1]
```diff
@@ -505,8 +505,11 @@ texture_clean_transparent (Clean transparent textures) bool false
 #    texture autoscaling.
 texture_min_size (Minimum texture size) int 64
 
-#    Experimental option, might cause visible spaces between blocks
-#    when set to higher number than 0.
+#    Use multi-sample antialiasing (MSAA) to smooth out block edges.
+#    This algorithm smooths out the 3D viewport while keeping the image sharp,
+#    but it doesn't affect the insides of textures
+#    (which is especially noticeable with transparent textures).
+#    A restart is required after changing this.
```
Comment (HybridDog) [inline builtin/settingtypes.txt:512]: Please do not remove the information that this is experimental. #6860

Comment (Calinou) [reply builtin/settingtypes.txt]: Done.

**Review by sfan5 — APPROVED** [review]

**Review by SmallJoker — APPROVED** [review]
LGTM

**Pull request merged**